git-extra/PKGBUILD

```diff
@@ -65,5 +65,6 @@ package() {
   install -m755 create-shortcut.exe $pkgdir/$mingwdir/bin
   install -m755 $startdir/git-prompt.sh $pkgdir/etc/profile.d
   install -m755 $startdir/aliases.sh $pkgdir/etc/profile.d
+  install -m755 $startdir/homepath.sh $pkgdir/etc/profile.d
```

Comment:
Maybe rename the file to something else? "homepath" sounds to me as if the path to home, i.e. the HOME variable, was being tampered with. Why not just call it "env.sh" or so to say that any environment overrides should be put into there?